libyul/optimiser/Metrics.cpp

```diff
@@ -70,8 +70,14 @@ size_t CodeWeights::costOf(Expression const& _expression) const
 		return functionCallCost;
 	else if (holds_alternative<Identifier>(_expression))
 		return identifierCost;
-	else if (holds_alternative<Literal>(_expression))
-		return literalCost;
+	else if (Literal const* literal = get_if<Literal>(&_expression))
+	{
+		// Avoid strings because they could be longer than 32 bytes.
```

Comment:
How can Yul string literals be more than 32 bytes?

Reply:
If it is a literal argument of a builtin.

Reply:
Ah, of course in verbatim.